queries.sql

```diff
@@ -0,0 +1,8 @@
+SELECT * FROM animals WHERE name LIKE '%mon'; -- Find all animals whose name ends in "mon".
+SELECT name FROM animals WHERE date_of_birth > '2016-01-01' AND date_of_birth < '2019-01-01'; -- List the name of all animals born between 2016 and 2019.
```

Comment:
- [ ] The animals born on the first day of 2016 and the last day of 2019 should also be included in the query. e.g.
```sql
SELECT name FROM animals WHERE date_of_birth >= '2016/01/01' AND date_of_birth <= '2019/31/12';
```